multipolygon.go

```diff
@@ -25,6 +25,19 @@ func (g *MultiPolygon) Area() float64 {
 	return doubleArea3(g.flatCoords, 0, g.endss, g.stride) / 2
 }
 
+// Empty returns whether the MultiPolygon is empty, or contains entirely empty Polygons.
+func (g *MultiPolygon) Empty() bool {
+	if g.geom3.Empty() {
+		return true
+	}
+	for i := 0; i < g.NumPolygons(); i++ {
+		if !g.Polygon(i).Empty() {
+			return false
+		}
+	}
+	return true
+}
```

Comment:
As with `MultiLineString.Empty` added in this PR, all the tests pass if we remove this function. Is there a case here where the fallback to `geom3.Empty` fails?

Reply:
nope! same work backwards step.